lib/viewing/ViewsFromServer.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+var $ = require("../application/jquery");
+
+var IDENTITY_ATTRIBUTE = "data-view-uid";
+
+var viewElements = {};
+
+var ViewsFromServer = {
+
+    IDENTITY_ATTRIBUTE: IDENTITY_ATTRIBUTE,
+
+    initialize: function() {
+        var $viewElements = $("[" + IDENTITY_ATTRIBUTE + "]");
+
+        $viewElements.each(function(i, viewElement) {
+            var uid = $(viewElement).attr(IDENTITY_ATTRIBUTE);
+
+            viewElements[uid] = viewElement;
+        });
+    },
```

Comment:
Is there any noticeable performance difference on initial load when you do this? The difference is a single $ query vs n $ queries where n is the number of views I assume?